packages/rest/test/integration/rest.server.integration.ts

```diff
@@ -52,6 +53,28 @@ describe('RestServer (integration)', () => {
     expect(server.url).to.be.undefined();
   });
 
+  it('parses query without decorated rest query params', async () => {
+    // This handler responds with the query object (which is expected to
+    // be parsed by Express)
+    function requestWithQueryHandler(handler: {
+      request: Request;
+      response: Response;
+    }) {
+      const {request, response} = handler;
+      response.json(request.query);
+      response.end();
+    }
+
+    // See https://github.com/strongloop/loopback-next/issues/2088
+    const server = await givenAServer({rest: {port: 0, host: '127.0.0.1'}});
```

Comment:
Please use `givenHttpServerConfig`, it will take care of setting the correct host & port for our CI.